### DISCUSSION TITLE
QNAP Container - GPU for transcoding

### DISCUSSION THREAD

**Discussion by RubbeeDuck** [jlesage/docker-handbrake#276] [Q&A]
Hello. 

I have installed on My QNAP over Container Station the Handbrake Container. 

I have activate the GPU under Device in the Settings. 

But when I Transcode something I can‘t see any difference between this Container with GPU and another one without GPU. 

Maybe someone can help me. 

Regards

**Comment by jlesage** [comment]
If you look at the container's log, you should see if QSV is correctly setup/usable.
